### PR TITLE
CFE-3238 service active state detection improved

### PR DESCRIPTION
**Pull request by basvandervlies**
Ticket: CFE-3238
Changelog: Title

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

File: lib/services.cf [hunk 1]
```diff
@@ -239,6 +239,10 @@ bundle agent standard_services(service,state)
       "service_enabled" -> { "CFE-2923" }
         expression => returnszero( "$(call_systemctl) is-enabled $(service) > /dev/null 2>&1", useshell);
       "service_active"  expression => reglist(@(systemd_service_info), "ActiveState=active");
+      "service_active"  or => {
```
Comment (nickanderson) [inline lib/services.cf]: Shouldn't the promise on line 241 be redacted since this performs the same check?

Comment (nickanderson) [reply lib/services.cf]: Or, can we consolidate them into a single call?
```
      "service_active"  expression => reglist(@(systemd_service_info), "ActiveState=(active|activating)");
```

**Comment by nickanderson** [comment]
Thanks @basvandervlies this PR is superseded by https://github.com/cfengine/masterfiles/pull/1673

**Pull request closed without merging**